docs/adrs/0361-wrapper-action.md

```diff
@@ -0,0 +1,73 @@
+# ADR 361: Wrapper Action
+
+**Date**: 2020-03-06
+
+**Status**: Pending
+
+## Context
+
+In addition to action's regular execution, action author may wants their action has a chance to participate in:
+- Job initialize  
```

Comment:
Why participate in job initialize?

- If for docker login, we can solve that a different way
- If for running stuff on host (for container job), conflicts with future policy scenarios.

We should do the `reserve-this-agent-across-multiple-jobs` feature instead.

Reply:
@chrispat for any other scenarios. 😄